utils/notifications.js

```diff
@@ -427,13 +428,26 @@ async function getParticipantsAndSendNotifications({ notificationSpec, cutoffTim
       let { emailRecordArray, emailPersonalizationArray, smsRecordArray } = notificationData[lang];
       if (emailPersonalizationArray.length > 0) {
         const emailBatch = {
-          from: {
-            name: process.env.SG_FROM_NAME || "Connect for Cancer Prevention Study",
-            email: process.env.SG_FROM_EMAIL || "donotreply@myconnect.cancer.gov",
-          },
-          subject: emailInSpec[lang].subject,
-          html: emailInSpec[lang].body,
-          personalizations: emailPersonalizationArray,
+            from: {
+                name:
+                    process.env.SG_FROM_NAME ||
+                    "Connect for Cancer Prevention Study",
+                email:
+                    process.env.SG_FROM_EMAIL ||
+                    "donotreply@myconnect.cancer.gov",
+            },
+            subject: emailInSpec[lang].subject,
+            html: emailInSpec[lang].body,
+            personalizations: emailPersonalizationArray,
+            tracking_settings: {
+                subscription_tracking: {
+                    enable: true,
+                    html:
+                        lang === "english"
+                            ? "<p><i>To unsubscribe from emails about Connect from the National Cancer Institute (NCI), <% click here %> .</i></p>"
+                            : "<p><i>Si desea darse de baja de Para cancelar la suscripción a los correos electrónicos sobre Connect del Instituto Nacional del Cáncer (NCI), <% haga clic aquí %> .</i></p>",
```

Comment:
Instead of using ternary operator here, accessing a predefined object containing the texts would be cleaner, something like `html: unsubscribeObj[lang] || unsubscribeObj.english`

Reply:
solved